matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/relation/RelationContent.kt

```diff
@@ -24,4 +24,5 @@ interface RelationContent {
     val eventId: String?
     val inReplyTo: ReplyToContent?
     val option: Int?
+    val isFallingBack: Boolean? // Thread fallback to differentiate replies within threads
```

Comment:
On the form I would use a `javadoc`. I do not like comment after the code

So something like
```kotlin
/**
 * doc...
 */
val isFallingBack: Boolean?
```

-> more readable and better integration with the IDE.

Also can you document a bit more what is the purpose of this `val`? What value `true` or `false` mean for this boolean? I have to confess that the name `isFallingBack` means nothing to me.

Reply:
This flag, was a last minute change that I was not aware about. More details are [here](https://github.com/matrix-org/matrix-spec-proposals/blob/gsouquet/threading-via-relations/proposals/3440-threading-via-relations.md#backwards-compatibility). I will add a more precise comment